P0009/P0009.md

```diff
@@ -1755,14 +1742,14 @@ M::is_always_strided()
                    For certain layout mappings it may not be feasible to determine whether every instance is strided.<i>— end note]</i>
 
 
-<b>24.7.�.3 Layout policy requirements [mdspan.layoutpolicy.reqmts]</b>
+<b>24.7.�.3 Layout mapping policy requirements [mdspan.layoutpolicy.reqmts]</b>
```

Comment:
It seems a bit inconsistent that the **[mdspan.layout.reqmts]** section is called "Requirements," but the **[mdspan.layoutpolicy.reqmts]** section is called "Layout mapping policy requirements."  Is the issue that there is no parent **[mdspan.layoutpolicy]** section, as there is with **[mdspan.layout]**?

Reply:
Yeah

Reply:
yeah